addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -42,6 +52,12 @@ FormRenderer.include({
         this.off('o_message_posted', this);
         owl.Component.env.bus.off('mail.thread:promptAddFollower-closed', this);
     },
+    /**
+     * @returns {Element|undefined} Scrollable Element
+     */
+    getScrollableElement() {
+        return $(".o_content") && $(".o_content")[0];
```

Comment:
This condition seems too generous, looking into the whole DOM. Shouldn't it be at least `this.$(` instead?

Reply:
We are unable to get the element in this. we have only channelId in this.